tool.go

```diff
@@ -404,15 +428,17 @@ func main() {
 					args = append(args, "-test.v")
 				}
 				status := "ok  "
-				start := time.Now()
-				if err := runNode(tempfile.Name(), args, pkg.Dir, options.Quiet); err != nil {
-					if _, ok := err.(*exec.ExitError); !ok {
-						return err
+				if !*compileOnly {
```

Comment:
Please transform this into an early exit and move it above the `var args []string` line.